src/firejail/profile.c

```diff
@@ -1502,8 +1505,11 @@ int profile_check_line(char *ptr, int lineno, const char *fname) {
 			arg_rlimit_sigpending = 1;
 		}
 		else if (strncmp(ptr, "rlimit-as ", 10) == 0) {
-			check_unsigned(ptr + 10, "Error: invalid rlimit in profile file: ");
-			sscanf(ptr + 10, "%llu", &cfg.rlimit_as);
+			cfg.rlimit_as = parse_arg_size(ptr + 10);
+			if ( cfg.rlimit_as == 0 ){
+				perror("Error: invalid rlimit-as size in profile file. use only non-negative numbers and k,m,g suffix for size");
```

Comment:
suggestion:
 perror("Error: invalid rlimit-as in profile file. Only use positive numbers and k,m or g suffix.");